_opensearch/cluster-manager-task-throttling.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Cluster manager task throttling
+nav_order: 70
+has_children: false
+---
+
+# Cluster manager task throttling
+
+For many cluster activities, such as defining a mapping or creating an index, data nodes submit tasks to the cluster manager. The cluster manager maintains a pending task queue for these tasks and runs them in a single-threaded environment. Sometimes data nodes may flood the cluster manager with too many tasks at the same time. When this happens, the number of tasks in the queue spikes; this affects the cluster manager performance, and may in turn affect the availability of the whole cluster.
```

Comment:
>The cluster manager maintains a pending task queue for these tasks and runs them in a single-

and executes them in ...

Reply:
>Sometimes data nodes may flood the cluster manager with too many tasks at the same time.

In the past, put-mappings or snapshot tasks have caused too much pile of pending tasks on cluster manager

Reply:
There is a need to build protection mechanism in the cluster manager itself.

Reply:
Hi @shwetathareja. Thanks for your suggestions. The word "executes" is on the list of words to avoid in our [style guide](https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md). The style guide suggests replacing it with the word "run".